content/blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc

```diff
@@ -0,0 +1,243 @@
+the best idea.
+
+.Screenshot displaying the list of newbie-friendly tickets in the Jenkins Jira
+image::/images/post-images/2019-05-30-becoming-contributor/list-newbie-tickets.png[]
+
+=== Selecting a ticket
+
+In my case, I spent some time reviewing the `newbie-friendly` tickets, until https://issues.jenkins-ci.org/browse/JENKINS-56477[I found one]
+that seemed interesting to me and also looked like something I would be able to fix:
+
+.Screenshot of the ticket I decided to work on
+image::/images/post-images/2019-05-30-becoming-contributor/jenkins-newbie-jira-ticket-selected.png[]
+
+=== Processes
+
+At this stage, when we have decided to take ownership of a ticket, it’s a good practice to let the rest of the community
+know that we are planning to start working on it. We can do so easily, by assigning the ticket to ourselves (see the
+“_Assign_” button below the ticket summary).
+
+Assigning the ticket to ourselves in the Jenkins Jira will allow any other contributors to know that we are planning to
+take care of the ticket; and in case they are also interested in contributing to it, they will know who to reach if they
+want to coordinate work or ask for status. That said, it is worth mentioning that assigning a ticket to yourself does
+not mean that other contributors cannot work on it from then onwards. Jenkins is an open-source project and anyone is
+welcome to create their own PRs, so anybody can propose their own solution to the ticket. But as you can guess, if the
+ticket is assigned to somebody, most people will probably reach the assignee before starting to work on it.
+
+Related to the former, it is also important to bear in mind that we should not postpone work on the ticket for too long
+once we have assigned the ticket to ourselves. Other potential contributors might be discarding the ticket because they
+see yourself assigned to it.
+
+Once we are about to actually start working on the ticket, it is also a good practice to click the “_Start Progress_”
+button. This action will change the status to “_In progress_”, signaling to the community that we are currently working
+on this particular ticket.
+
+== Setting up the necessary stuff on our computer
+
+=== Configuring, installing and testing
+
+As described in the https://jenkins.io/blog/2019/03/29/becoming-contributor-intro/[first article of this journey], the
+initial step to start contributing to a particular repository is to fork it to our GitHub account, and then clone it to
+our local computer.
+
+As usual, in the Jenkins core repository the https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md[CONTRIBUTING file]
+describes the necessary steps to get the repository working locally. This includes installing the necessary development
+tools: Java Development Kit (_http://openjdk.java.net/[OpenJDK] is the recommended choice_), Maven and any IDE supporting
```

Comment:
I find it easier to locate a JDK to download from the Adopt OpenJDK download site.  Others in the Jenkins project also seem to prefer that site rather than using openjdk.net.

```suggestion
tools: Java Development Kit (_https://adoptopenjdk.net/[OpenJDK] is the recommended choice_), Maven and any IDE supporting
```